Library/Formula/ack.rb

```diff
@@ -1,4 +1,5 @@
 class Ack < Formula
+  desc "A search tool like grep, but optimized for programmers"
   homepage "http://beyondgrep.com/"
```

Comment:
My own personal strong preference is that we put the `desc` as the last line of the initial block, i.e. under either the `sha` or `revision`. For some reason, having it as the top line is making me very uncomfortable, but I'm just weird.